### PR TITLE
Add basic support to run unittests in GitHub Actions.

### PR DESCRIPTION
**Pull request by sodul**
Includes a .gitignore file so we do not add unwanted files accidentally.
A very basic unittest so the module can at least be loaded, and a starting point for more unittests.
Clarify the macOS support in the README.
Add more pypi classifier for the project.
Update the repository url to the new home.

### CODE REVIEW

**Comment by sodul** [comment]
This does not run the existing integration tests. They are not python 3 compatible and require a database to connect to.

The goal here is to add some bare checks to ensure that pypyodbc.py is importable by currently active versions of python and will capture obvious syntax errors.

Longer term I will look into adding a MSSQL container to run the tests against and update the existing tests to be compatible with python 3.

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# GitHub action to run on every Pull Request.
+
+name: CI
+
+on:
+  push:
+    branches: [ master ]
```
Comment (rolweber) [inline .github/workflows/ci.yaml:7]: In this repo, the default branch is `main`.

**Review by rolweber — COMMENTED** [review]
Thanks a lot! Only one minor finding.

**Review by rolweber — APPROVED** [review]
lgtm

**Comment by rolweber** [comment]
I'll give @braian87b a few days to review as well.

**Pull request merged**